draft-mouris-cfrg-mastic.md

```diff
@@ -237,9 +285,98 @@ This document uses the following terms as defined in {{!VDAF}}:
+|:--------------------|:------------------------------------------------------|:-----------------------------------------------------------|
+| `SEED_SIZE: int`    | the size (in bytes) of the XOR seed                   | 32 for `XofTurboShake128` or 16 for `XofFixedKeyAes128`    |
+{: #xof-params title="XOF parameters."}
+
+
+Mastic uses Fully Linear Proof (FLP) Systems as specified in {{Section 7.1 of
+  !VDAF}}. The following FLP API functions referenced in this document are as
+  defined therein:
+
+  * `flp.decide(verifier: list[F]) -> bool` is a deterministic decision algorithm
+    of a verifier that takes as input a verifier message and outputs a Boolean
+    indicating if the measurement from which it was generated is valid.
+  * `flp.decode(output: list[F], num_measurements: int) -> AggResult` maps a sum
+    of aggregate shares to an aggregate result.
+  * `flp.encode(measurement: Measurement) -> list[F]` encodes a raw measurement
+    as a vector of field elements. The return value MUST be of length `MEAS_LEN`.
+  * `flp.prove(meas: list[F], prove_rand: list[F], joint_rand: list[F]) ->
+    list[F]` is a prover's deterministic proof-generation algorithm. Its inputs
+    are the encoded measurement, the "prover randomness" `prove_rand`, and the
+    "joint randomness" `joint_rand`. The prover randomness is used only by the
+    prover, but the joint randomness is shared by both the prover and verifier.
+  * `flp.query(meas: list[F], proof: list[F], query_rand: list[F], joint_rand:
+    list[F], num_shares: int) -> list[F]` is a verifier's query algorithm. This
+    is used to "query" the measurement and proof. The result of the query (i.e.,
+    the output of this function) is called the "verifier message". In addition
+    to the measurement and proof, this algorithm takes as input the query
+    randomness `query_rand` and the joint randomness `joint_rand`. The former
+    is used only by the verifier. Here, num_shares specifies how many shares
+    were generated.
+  * `flp.truncate(meas: list[F]) -> list[F]` maps an encoded measurement (e.g.,
+    the bit-encoding of the measurement) to an aggregatable output (e.g., the
+    singleton vector containing the measurement). The length of the input MUST
+    be `MEAS_LEN` and the length of the output MUST be `OUTPUT_LEN`.
+
+We reference the following FLP parameters in this document:
+
+  | Parameter           | Description                                           | Value               |
+  |:--------------------|:------------------------------------------------------|:--------------------|
+  | `OUTPUT_LEN: int`   | the length of the aggregatable output                 | to be defined       |
+  | `MEAS_LEN: int`     | the length of the encoded measurement                 | to be defined       |
+  | `field: type[F]`    | class object for the field ({{Section 6.1 of !VDAF}}) | set by constructor  |
```

Comment:
This is only a subset of the FLP parameters.

Why  list `field: type[F]` here and in https://github.com/jimouris/draft-mouris-cfrg-mastic/pull/89/files#r1823348916?